tests/tamr_client/datasets/test_dataframe.py

```diff
@@ -15,7 +15,10 @@ def test_upsert():
     dataset = utils.dataset()
 
     url = tc.URL(path="datasets/1:updateRecords")
-    updates = utils.records_to_updates(_records_json)
+    updates = [
+        tc.record._create_command(record, primary_key_name="primary_key")
+        for i, record in enumerate(_records_json, start=1)
```

Comment:
`i` is unused, so you no longer need the `enumerate`:

```suggestion
        for record in _records_json
```

Same applies to other usages of `_create_command`, `_delete_command` in the test files.